app/equipment.py

```diff
@@ -0,0 +1,36 @@
+class Armor:
+    def __init__(self, armour: list) -> None:
+        self.armour = armour
+
+    def apply_armor(self) -> int:
+        return sum([part["protection"] for part in self.armour])

```

Comment:
you can remove square brackets here, it would be more readable

Reply:
Oh, yes! Thank you!